libs/providers/flagd/src/lib/service/grpc/service.ts

```diff
@@ -0,0 +1,111 @@
+import {IService} from '../IService'
+import {ServiceClient} from '../../../proto/ts/schema/v1/schema.client'
+import * as grpc from '@grpc/grpc-js';
+import {
+    EvaluationContext,
+    ResolutionDetails
+} from '@openfeature/nodejs-sdk'
+import {
+    ResolveBooleanRequest,
+    ResolveStringRequest,
+    ResolveNumberRequest,
+    ResolveObjectRequest
+} from '../../../proto/ts/schema/v1/schema'
+import { RpcError } from '@protobuf-ts/runtime-rpc'
+import { Struct } from '../../../proto/ts/google/protobuf/struct'
+import client from './client'
+
+export class GRPCService implements IService {
+    client: ServiceClient;
+    constructor(host?: string, port?: number, credentials?: grpc.ChannelCredentials) {
+        this.client = new client(host, port, credentials)
+    }
+
+
+    async ResolveBoolean(flagKey: string, defaultValue: boolean, context: EvaluationContext): Promise<ResolutionDetails<boolean>> {
+      // eslint-disable-next-line prefer-const
```

Comment:
why do you have to disable eslint here ?

Reply:
removed in latest commit